pyproject.toml

```diff
@@ -86,7 +86,10 @@ numpy = [
 optional = true
 
 [tool.poetry.group.docs.dependencies]
-Sphinx = ">=5.0.1"
+Sphinx = [
+  {version = ">=7.1.2,<7.2.0", python = ">=3.8,<3.9"},
+  {version = ">=7.2.6", python = ">=3.9,<3.13"},
```

Comment:
FYI, you don't actually need to update the lower bound here, `>=5.0.1,<7.2.0` and `>=5.0.1` would also work.